### PR TITLE
apply purestake licensing and copyright

### LINKED ISSUE #11: Update licensing
This should be licensed to Purestake and GPL 3 The headers need updated (or ideally removed, the gpl only encourages their use it does not require them). And make sure the repo license is clear.

### PR DESCRIPTION
**Pull request by librelois**
This PR also add a CI job to ensure that all rust files have copyright.

Closes #11

### CODE REVIEW

File: nimbus-consensus/src/import_queue.rs [hunk 1]
```diff
@@ -1,18 +1,18 @@
-// Copyright 2019 Parity Technologies (UK) Ltd.
-// This file is part of Cumulus.
+// Copyright 2019-2021 PureStake Inc.
+// This file is part of Moonbeam.
```
Comment (JoshOrndorff) [inline nimbus-consensus/src/import_queue.rs]:
```suggestion
// This file is part of Nimbus.
```

File: nimbus-consensus/src/import_queue.rs [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-// Cumulus is free software: you can redistribute it and/or modify
+// Moonbeam is free software: you can redistribute it and/or modify
```
Comment (JoshOrndorff) [inline nimbus-consensus/src/import_queue.rs]:
```suggestion
// Nimbus is free software: you can redistribute it and/or modify
```

File: nimbus-consensus/src/import_queue.rs (hunk 1 above)
Comment (JoshOrndorff) [reply nimbus-consensus/src/import_queue.rs]: I don't have strong feelings here, but I think libraries usually use their own name here as opposed to the flagship product that uses them.

**Review by JoshOrndorff — APPROVED** [review]
Basically good. Thank you!

You can take my suggestion or not, either is ok :)

File: nimbus-consensus/src/import_queue.rs (hunk 1 above)
Comment (librelois) [reply nimbus-consensus/src/import_queue.rs]: done

File: nimbus-consensus/src/import_queue.rs (hunk 2 above)
Comment (librelois) [reply nimbus-consensus/src/import_queue.rs]: Done

**Comment by JoshOrndorff** [comment]
I pushed one commit with a few more updates. It seems like some files are still missing licenses entirely. For example the `parachain-template/runtime/src/lib.rs`.

**Comment by librelois** [comment]
> I pushed one commit with a few more updates. It seems like some files are still missing licenses entirely. For example the `parachain-template/runtime/src/lib.rs`.

Yes I purposely excluded the parachain-template folder from the copyright check because this template is unlicensed, and I thought we wanted it to remain unlicensed.

**Comment by JoshOrndorff** [comment]
Ohh, I see. Yeah, that makes sense to me. The account set is my own code too, and I'm fine removing the license there to match the rest of the template.

**Review by JoshOrndorff — APPROVED** [review]
Merge when you are ready :)

**Pull request merged**